src/messages/Message.cpp

```diff
@@ -48,6 +48,13 @@ SBHighlight Message::getScrollBarHighlight() const
             ColorProvider::instance().color(ColorType::FirstMessageHighlight),
             SBHighlight::Default, false, true);
     }
+    else if (this->flags.has(MessageFlag::ElevatedMessage))
+    {
+        return SBHighlight(ColorProvider::instance().color(
+                               ColorType::ElevatedMessageHighlight),
+                           SBHighlight::Default, false, true);
+    }
+
```

Comment:
This area is the problematic code for the last issue
Currently the `enableFirstMessageHighlight` setting is disabling both First Messages and Elevated Messages.

Setting line 49 from `SBHighlight::Default, false, true);` to `SBHighlight::Default, true);` changed the issue to `enableFirstMessageHiglight` enable/disabling Elevated Messages scrollbar and `enableRedeemMessage` enable/disabling First Message scrollbar. `